src/services/WolfDen.Domain/Entity/LOP.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace WolfDen.Domain.Entity
+{
+    public class LOP
+    {
+        public int Id { get; private set; }
+        public int Month { get;private set; }

```

Comment:
Int won't suffice

Reply:
is enum ok?

Reply:
will that work ?

Reply:
actually in order to store the lop days in a month,a column for year too is required.
So that we can fetch the lopcount of a particular person of that month using the year and month which is passed as number from the frontend

Reply:
can you give example of the no.

Reply:
janaury is passed as 0,feb as 1 ....

Reply:
yes otherwise how can you identify months in different years